components/ui/Table.tsx

```diff
@@ -508,7 +508,6 @@ const Table = observer(
 
             {onPaginate ? (
               <Paginator
-                // onIndexChanged={handleIndexChange}
```

Comment:
Is this needed?

Reply:
No, it's a leftover from pagination where you could switch directly to page number, without infinite scroll etc.